iiasa/message_ix
# Improve tutorials

Improve the tutorials by fixing some function calls, numbers, and typos.

This PR appears way larger than it actually is. Most of the deletions originate from an apparently new way of jupyter/matplotlib to store svg files. Rather than in multiple lines, they are collapsed to single lines, releasing astronomical numbers of lines of storage space. To my eyes, the images in e.g. [tutorial/Austrian_energy_system/austria_multiple_policies-answers.ipynb](https://github.com/iiasa/message_ix/compare/main...glatterf42:message_ix:improve_tutorials?expand=1#diff-4928e70fa22996182fd344aabceae83a0150ccbb6d8796ed612c9abdaa4c2000) (which contains the vast majority of deletions) look the same as before.
One other main thing is to replace \text{} in math mode with \mathrm{}. As stated in mathjax/MathJax/issues/1770, this ensures correct behaviour of text and underscores in Latex math environments.



## How to review

To review, please 
- Check that equations render correctly
- Check that images look the same
- Answer any of the questions in the conversations below 



## PR checklist
